qa/tasks/cephfs/fuse_mount.py

```diff
@@ -304,15 +318,15 @@ def umount(self, cleanup=True):
                     """).format(self._fuse_conn))
                     self._fuse_conn = None
 
-                stderr = StringIO()
-                # make sure its unmounted
-                try:
-                    self.client_remote.run(args=['sudo', 'umount', '-l', '-f',
-                                                 self.mountpoint],
-                                           stderr=stderr, timeout=(60*15), omit_sudo=False)
-                except CommandFailedError:  
-                    if self.is_mounted():   
-                        raise
+            stderr = StringIO()
```

Comment:
Not sure. I remember hitting few issues with FUSE mounts but I am not sure if this is part of the fixes that I wrote since FUSE errors were discovered by me when I had started with testing with teuthology.

Reply:
Ran FUSE tests locally after reversing this patch, tests ran successfully. This must be a mistake on my part. Thanks for pointing out.